src/particles/diagnostics/DiagnosticOutput.H

```diff
@@ -21,7 +21,6 @@ namespace impactx::diagnostics
      */
     enum class OutputType
     {
-        PrintParticles, ///< ASCII diagnostics, for small tests only
         PrintNonlinearLensInvariants, ///< ASCII diagnostics for the IOTA nonlinear lens, for small tests only
         PrintRefParticle, ///< ASCII diagnostics, for small tests only
```

Comment:
Not sure why the documentation says "for small tests only" here.  The write to ASCII for the ref particle data should be a small cost, independent of the numerical resolution.  It is valuable data to have for any test involving a moderately complex beamline.

Reply:
True, that was probably copy-pasted.
```suggestion
        PrintRefParticle, ///< ASCII diagnostics
```